src/trackerGlobalMixin.js

```diff
@@ -224,6 +224,10 @@ export default {
       if (progressStore.copy('progress/quest_complete', quest.id)) {
         return 1
       }
+      // If the player level is below the required quest level, then the quest is not available
+      if(progressStore.get("progress/level") < quest.require.level) {
+        return -1
+      }
```

Comment:
Should the check be moved to the end, right before the return 0? The difference is there might be a case where a quest is blocked, but below the player's level. If that's the case, then the player will see it as locked, and then when they reach that level they will see it as blocked. Would this even ever happen?